cmd/minishift/cmd/start.go

```diff
@@ -588,10 +587,10 @@ func validateOpenshiftVersion() {
 }
 
 func setSubscriptionManagerParameters() {
-	cluster.RegistrationParameters.Username = viper.GetString(configCmd.Username.Name)
-	cluster.RegistrationParameters.Password = viper.GetString(configCmd.Password.Name)
-	cluster.RegistrationParameters.GetUsernameInteractive = getUsernameInteractive
-	cluster.RegistrationParameters.GetPasswordInteractive = getPasswordInteractive
+	minishiftCluster.RegistrationParameters.Username = viper.GetString(configCmd.Username.Name)
+	minishiftCluster.RegistrationParameters.Password = viper.GetString(configCmd.Password.Name)
+	minishiftCluster.RegistrationParameters.GetUsernameInteractive = getUsernameInteractive
+	minishiftCluster.RegistrationParameters.GetPasswordInteractive = getPasswordInteractive
```

Comment:
Pretty sure we can now even reduce the code slightly further... But some other time maybe?